<launch>
  <arg name="output" default="screen"/>

  <node pkg="predicator_collision"
    name="predicator_robot_interaction_node"
    type="predicator_robot_interaction_node"
    output="$(arg output)">

    <rosparam param="description_list">
      - /gazebo/barrett_manager/robot_description
      - /gazebo/w2barrett_manager/robot_description
      - /stage_model
    </rosparam>

    <rosparam param="joint_state_topic_list">
      - /gazebo/barrett_manager/joint_states
      - /gazebo/w2barrett_manager/joint_states
    </rosparam>

    <param name="verbosity" value="1"/>
    <param name="world_frame" value="/world"/>
  </node>
</launch>
